pom.xml

```diff
@@ -92,6 +93,7 @@
         <nacos.version>2.2.1</nacos.version>
         <!-- Spring -->
         <spring.framework.version>5.2.9.RELEASE</spring.framework.version>
+        <spring6.framework.version>6.0.8</spring6.framework.version>
```

Comment:
Can our SDK be used normally when referenced in the Spring project of 5. x

Reply:
Yes, it can